brownie-config.yml

```diff
@@ -14,11 +18,43 @@ dependencies:
 # path remapping to support imports from GitHub/NPM
 compiler:
   solc:
-    version: 0.6.12
+    version:
     remappings:
       - "@yearnvaults=yearn/yearn-vaults@0.4.3"
```

Comment:
make sure to use 0.4.3-1

Reply:
I will update. The tricrypto strats in arb and ftm that re recently deployed were using 0.4.3. Are we ok, or should we update those to 0.4.3-1 as well?

Reply:
I think 0.4.3-1 is a little nicer, all it adds is events that the yearn-watch guys found helpful. and on sidechains/L2s we really don't mind a little extra gas for events imo